proxy/http2/Http2ConnectionState.h

```diff
@@ -38,7 +38,7 @@ enum Http2SendADataFrameResult {
   HTTP2_SEND_A_DATA_FRAME_DONE       = 3,
 };
 
-enum Http2ShutdownState { NOT_INITIATED, INITIATED, IN_PROGRESS };
+enum Http2ShutdownState { NOT_PLANNED, NOT_INITIATED, INITIATED, IN_PROGRESS };
```

Comment:
These enums should be renamed to be consistent with the rest of the code. I think that all you need are:

```C
enum Http2ShutdownState {
    HTTP2_SHUTDOWN_NONE,
    HTTP2_SHUTDOWN_BEGIN,
    HTTP2_SHUTDOWN_INPROGRESS
};
```